src/erl.rs

```diff
@@ -33,15 +39,15 @@ pub fn generate_erlang(analysed: &[Analysed]) -> Vec<OutputFile> {
     {
         let gen_dir = source_base_path
             .parent()
-            .unwrap()
+            .gleam_expect("The source base path terminates in a root or prefix")
             .join(project::OUTPUT_DIR_NAME)
             .join(origin.dir_name());
         let erl_module_name = name.join("@");
 
-        for (name, text) in records(ast).into_iter() {
+        for (name, text) in &records(ast) {
             files.push(OutputFile {
                 path: gen_dir.join(format!("{}_{}.hrl", erl_module_name, name)),
-                text,
+                text: text.clone(),
```

Comment:
Let's undo this change to the loop, is it slower now as it has to clone the text each time.

Reply:
https://rust-lang.github.io/rust-clippy/master/index.html#explicit_into_iter_loop
This was the reason... If you want me to change it, I'll add a global allow for this lint or allow for this function.

Reply:
This seems to be a buggy lint as the suggested alternative is slower. Let's not enable this one